src/mjolnir/convert_transit.cc

```diff
@@ -413,9 +410,7 @@ std::list<PointLL> GetShape(const PointLL& stop_ll,
                             const float orig_dist_traveled,
                             const float dest_dist_traveled,
                             const std::vector<PointLL>& trip_shape,
-                            const std::vector<float>& distances,
-                            const std::string& origin_id,
-                            const std::string& dest_id) {
```

Comment:
these aren't used in this function (which looks proper). these are the onestop_id for origin/dest and this was the only time onestop_ids were used in convert_transit. it seems onestop_ids for the stop_pairs are only relevant for ingest_transit for the purpose of stitching.

Reply:
(actually they were used, but only for printing when smth went wrong in `GetShape()`, I replaced that with both stop's lon/lat)